### ISSUE TITLE
About the mask.

### ISSUE THREAD

**Issue by CodeLHY** [GodZarathustra/StablePose#2]
is the mask, for example "\T-LESS\test_primesense\000002\mask_visib\000003_000000.png" the  result of your detection network? I wonder how can I get this mask in my own dataset.

**Comment by yifeishi** [comment]
Yes, I guess so. Using any detector, such as maskrcnn, could solve your problem

**Comment by CodeLHY** [comment]
I mean, is the mask is predicted by your detection network? Or in other words, how did you get the mask of every single image?

**Comment by GodZarathustra** [comment]
> I mean, is the mask is predicted by your detection network? Or in other words, how did you get the mask of every single image?

Thanks for your interest. No, the dataset released here is the official version, if you want to get the object detection results, please refer to the code here https://github.com/kirumang/Pix2Pose

**Comment by CodeLHY** [comment]
Thanks for your prompt reply. Can you release the detection result that you used in this work? I think this is a necessary to reproduce this work. And it seems that in this released code, you are using the ground truth mask, not the mask predicted by your detection network.
